dts/bindings/serial/ambiq,uart.yaml

```diff
@@ -0,0 +1,20 @@
+# Copyright (c) 2023 Antmicro <www.antmicro.com>
+# SPDX-License-Identifier: Apache-2.0
+
+description: Ambiq UART controller (PL011 compatible)
+
+compatible: "ambiq,uart"
+
+include: ["arm,pl011.yaml", pinctrl-device.yaml]
+
+properties:
+  pinctrl-0:
+    required: true
+
+  pinctrl-names:
+    required: true
+
+  pwrcfg:
+    type: phandle-array
+    specifier-space: pwrcfg
+    description: Power configuration
```

Comment:
note: you can likely have a common file defining this, so all ambiq IP peripherals can re-use it

Reply:
Sure. I moved it to `ambiq-pwrcfg.yaml`